### ISSUE TITLE
importcontract fails with returntype >= 16

### ISSUE THREAD

**Issue by igormcoelho** [CityOfZion/neo-python#217]
Perhaps I misunderstood something, but I cannot import a contract with return type >= 16 (example, Void = ff = 255).
What happens is that "def GatherContractDetails(function_code, prompter)" will call "return generate_deploy_script(function_code.Script, name, version, author, email, description,                                   function_code.ContractProperties, ord(function_code.ReturnType), function_code.ParameterList)" on line 199 of LoadSmartContract.py, and since returntype is passed in ord() function, it will be set as an int < 256.
Function "def generate_deploy_script" on line 204 of LoadSmartContract.py will receive return_type and push it "sb.push(return_type)" at line 220. Remember that return_type is always "int" in this case (because of ord() function called previously), but push function in ScriptBuilder.py will enter "if type(int) is int or ..." and will fail "else" command: return self.push(binascii.hexlify(data.ToByteArray())) because "int" cannot have ToByteArray.
I believe this code only works for return types < 16, such as ByteArray 0x05, Boolean 0x01, etc (because it wont  enter "else" condition), and I think an easy solution is to force return value to be BigInteger (in line 220 of LoadSmartContract.py). It will become: "sb.push(BigInteger(return_type))". 
I can make a pull request, but since it seems a rather complicated issue, I prefer to ask for opinions first :)

**Comment by localhuman** [comment]
Yes, this has been fixed.

**Comment by vncoelho** [comment]
It is not in the master yet.

The error is still happening here.

**Comment by igormcoelho** [comment]
The problem was solved indeed, I can confirm it. The solution was the adoption of a bytearray( ff ), which is quite similar to my older proposition using BigInteger (ff)... that's what we have used successfully in neocompiler.io for the past few months.
Vitor noticed that another issue indeed was happening, related to #422, but not in deploy time... it seemed databased was being blocked when a wrong readbyte was performed in a further invocation.. but that seems to be solved in 'development' branch now (not in current master https://github.com/CityOfZion/neo-python/commit/42e0bb41866d0b2c7aa3ee460a8f8ce3ae2f1f22).

As a solution, we are using development branch now, even for production :)
